i18n/en.json

```diff
@@ -5843,6 +5843,10 @@
     "id": "app.post.get_flagged_posts.app_error",
     "translation": "Unable to get the flagged posts."
   },
+  {
+    "id": "app.post.get_new_team_members_since.app_error",
+    "translation": " "
```

Comment:
Should we fill this translation entry?

Reply:
It's not being used by the front-end so per can leave it blank to avoid unnecessary work for the translators.